openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -98,14 +98,14 @@ Issuer-Holder-Verifier Model:
 Holder Binding:
 :  Ability of the Holder to prove legitimate possession of a Verifiable Credential.
 
-Cryptographic Key Binding:
-:  Ability of the Holder to prove legitimate possession of a Verifiable Credential by proving control over the same private key used during issuance and presentation. The mechanism might depend on the Credential Format. For example, in IETF SD-JWT VC [@I-D.ietf-oauth-sd-jwt-vc], the Issuer can enable key binding by including a public key or a reference to a public key, matching a private key controlled by the Holder, in the `cnf` claim.
+Cryptographic Holder Binding or Cryptographic Key Binding:
+:  Ability of the Holder to prove legitimate possession of a Verifiable Credential by proving control over the same private key during the issuance and presentation. Mechanism might depend on the Credential Format. For example, in IETF SD-JWT VC [@I-D.ietf-oauth-sd-jwt-vc], the Issuer can enable this binding by including a public key or a reference to a public key that matches to the private key controlled by the Holder.
 
-Claims-based Binding:
-:  Ability of the Holder to prove legitimate possession of a Verifiable Credential by proofing certain claims, e.g., name and date of birth, for example, by presenting another Verifiable Credential. Claims-based Holder Binding allows long-term, cross-device use of a Credential as it does not depend on cryptographic key material stored on a certain device. One example of such a Verifiable Credential could be a Diploma.
+Claims-based Holder Binding:
```

Comment:
I'm okay with this change, but shouldn't we update the use of "Claims-based Binding" in the rest of the doc to say "Claims-based Holder Binding" too?